src/package-manager.ts

```diff
@@ -0,0 +1,35 @@
+import { existsSync } from 'node:fs'
+import { select, confirm } from '@inquirer/prompts'
+import { PackageManager, LOCK_FILES, PACKAGE_MANAGERS } from './constants'
+
+export default async function choosePackageManager(): Promise<PackageManager> {
```

Comment:
The reason I made it default is because there's an active ESLint rule:
> Prefer default export on a file with single export. eslint[import/prefer-default-export](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/prefer-default-export.md)